projects/demo/src/components/items-handlers/index.html

```diff
@@ -2,10 +2,7 @@
     <td colspan="3">
         <span tuiTitle>
             <strong>ItemsHandlers</strong>
-            <span tuiSubtitle>
-                Applied as a host directive to
-                <code>&lt;tui-textfield /&gt;</code>
-            </span>
+            <span tuiSubtitle>Applied as a host directive</span>
```

Comment:
I'm concerned that it can not obvious for user where to put handlers.

```html
<tui-textfield>
    <input [disabledItemHandler]="..." />
</tui-textfield>
```

or 

```html
<tui-textfield [disabledItemHandler]="...">
    <input />
</tui-textfield>
```